dbux-data/src/dataProviderUtil.js

```diff
@@ -303,22 +306,22 @@ export default {
   // },
 
   /**
-   * NOTE: Trace's value could be `undefined` or `null`, make sure to use `dp.util.doesTraceHaveValue` before this.
+   * Get the value of trace, return `undefined` if trace has no value
```

Comment:
Comment:
```
@return Value of given trace. If value is `undefined`, it could mean that the `value` is actually `undefined`, or, in case of traces that are not expressions, that there is no value.

```